databricks/sdk/__init__.py

```diff
@@ -58,6 +60,11 @@
 
 
 class WorkspaceClient:
+    _config: client.Config
+
+    @property
+    def config(self):
+        return deepcopy(self._config)
```

Comment:
i'm not sure it's a good thing. `w.config.authenticate()` should reuse the same refresh token

Reply:
Riggghttt. Completely forgot about this. We can atleast restrict the initial config form being passed around. If they directly reuse config from client, or edit it, atleast they know why stuff is changing.